src/argowrapper/workflows/argo_workflows/gwas.py

```diff
@@ -190,7 +195,7 @@ def generate_argo_workflow(self):
         return super()._to_dict()
 
     @staticmethod
-    def interpret_gwas_workflow_error(step_name: str, step_log: str) -> str:
+    def interpret_gwas_workflow_error(step_name: str, step_log: List) -> str:
```

Comment:
not sure if step_log is List....

Reply:
Nice catch, not sure why I changed it and where it complained (or was it accidental).